server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/DumbBlobStoreCacheContract.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.james.blob.cassandra.cache;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.james.blob.api.BlobId;
+import org.junit.Test;
```

Comment:
The test error is because you are relying on a junit 4 annotation in a junit 5 test class.

Should be:

```
import org.junit.jupiter.api.Test;
```